client/src/app/project-form/project-ingest-schema.json

```diff
@@ -14,17 +14,17 @@
       "guidelines": "Enter the ideal date when you would have accessions available for your project. We generally require a minimum of 2 weeks to provide an accession. We will endeavour to provide your accessions by the date given",
       "type": "string",
       "format": "date-time",
-      "user_friendly": "Accession Time Frame"
+      "user_friendly": "Accession time frame"
     },
     "dataAccess": {
       "$schema": "http://json-schema.org/draft-07/schema#",
-      "description": "Data Access",
-      "title": "Data Access",
+      "description": "Data access",
```

Comment:
I would argue that if we have a property called dataAccess, we can have the default title as "Data access" calculated programmatically, removing a lot of boilerplate code from this file.

Reply:
@amnonkhen I think you are right, but I would do it in another ticket. That change is a bit too much change in context of the current ticket. What do you think?

Reply:
yeah, i need to check where i could put that logic. I'll go with this for now!

Reply:
@amnonkhen is it possible to convert the requested changes as a comment? it's not allowing me to merge unless I merge it as admin  🤔

Reply:
`Merging can be performed automatically once the requested changes are addressed.`